Jaeger query

Openshift integration tests are failing with error:

[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.05 s <<< FAILURE! - in me.snowdrop.istio.dt.OpenshiftIT
[ERROR] tracingTest  Time elapsed: 0.762 s  <<< ERROR!
java.lang.NullPointerException
	at me.snowdrop.istio.dt.OpenshiftIT.getBaseUrlByRouteName(OpenshiftIT.java:135)
	at me.snowdrop.istio.dt.OpenshiftIT.setup(OpenshiftIT.java:40)

because of bad route name: 
https://github.com/snowdrop/istio-distributed-tracing-example/blob/8573a9f478c71ab985c8209c885efaaf6e7ba4d8/tests/src/test/java/me/snowdrop/istio/dt/OpenshiftIT.java#L29

The correct istio jaeger route name is jaeger.

--- a/tests/src/test/java/me/snowdrop/istio/dt/OpenshiftIT.java
+++ b/tests/src/test/java/me/snowdrop/istio/dt/OpenshiftIT.java
@@ -26,7 +26,7 @@ import io.restassured.response.Response;
 public class OpenshiftIT {
 
     private static final String ISTIO_NAMESPACE = "istio-system";
-    private static final String JAEGER_QUERY_NAME = "jaeger-query";
+    private static final String JAEGER_QUERY_NAME = "jaeger";
     private static final String ISTIO_INGRESS_GATEWAY_NAME = "istio-ingressgateway";
 
     @Inject

